mapping_files/item_statuses.tsv

```diff
@@ -1,4 +1,4 @@
-CURRENTLOCATION	folio_name
+legacy_code	folio_name
```

Comment:
Shouldn't this match the value of `legacy_field` in item_mapping.json?

Reply:
Per Jeremy: item_mapping.json is separate from the item_status.tsv column names. The folio_migration_tools expect legacy_field as a column name for item status and will error if it is different.